website/docs/r/media_streaming_policy.html.markdown

```diff
@@ -0,0 +1,223 @@
+
+---
+
+* `common_encryption_cbcs` - (Optional) A `common_encryption_cbcs` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+* `common_encryption_cenc` - (Optional) A `common_encryption_cenc` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+* `default_content_key_policy_name` - (Optional) Default Content Key used by current Streaming Policy. Changing this forces a new Streaming Policy to be created.
+
+* `no_encryption_enabled_protocols` - (Optional) A `no_encryption_enabled_protocols` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+---
+
+A `common_encryption_cbcs` block supports the following:
+
+* `default_content_key` - (Optional) A `default_content_key` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+* `drm_fairplay` - (Optional) A `drm_fairplay` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+* `enabled_protocols` - (Optional) A `enabled_protocols` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+---
+
+A `common_encryption_cenc` block supports the following:
+
+* `default_content_key` - (Optional) A `default_content_key` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+* `drm_playready` - (Optional) A `drm_playready` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+* `drm_widevine_custom_license_acquisition_url_template` - (Optional) TODO. Changing this forces a new Streaming Policy to be created.
+
+* `enabled_protocols` - (Optional) A `enabled_protocols` block as defined below. Changing this forces a new Streaming Policy to be created.
+
+---
+
+A `default_content_key` block supports the following:
+
+* `label` - (Optional) Label can be used to specify Content Key when creating a Streaming Locator. Changing this forces a new Streaming Policy to be created.
+
+* `policy_name` - (Optional) Policy used by Default Key. Changing this forces a new Streaming Policy to be created.
+
+---
+
+A `drm_fairplay` block supports the following:
+
+* `allow_persistent_license` - (Optional) All license to be persistent or not. Changing this forces a new Streaming Policy to be created.
+
+* `custom_license_acquisition_url_template` - (Optional) Template for the URL of the custom service delivering licenses to end user players. Not required when using Azure Media Services for issuing licenses. The template supports replaceable tokens that the service will update at runtime with the value specific to the request. The currently supported token values are {AlternativeMediaId}, which is replaced with the value of StreamingLocatorId.AlternativeMediaId, and {ContentKeyId}, which is replaced with the value of identifier of the key being requested. Changing this forces a new Streaming Policy to be created.
```

Comment:
these likely want to be quoted so that they `stand out` in the description:

```suggestion
* `custom_license_acquisition_url_template` - (Optional) Template for the URL of the custom service delivering licenses to end user players. Not required when using Azure Media Services for issuing licenses. The template supports replaceable tokens that the service will update at runtime with the value specific to the request. The currently supported token values are `{AlternativeMediaId}`, which is replaced with the value of `StreamingLocatorId.AlternativeMediaId`, and `{ContentKeyId}`, which is replaced with the value of identifier of the key being requested. Changing this forces a new Streaming Policy to be created.
```